kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -40,7 +54,7 @@ window.Exercises = {
         render: function() {}
     },
     incompleteStack: [0]
-};
+}, Backbone.Events);
```

Comment:
Add Backbone.Events to the Exercises object so we can listenTo it rather than doing $.bind to respond to its events.